src/Microsoft.TestPlatform.CrossPlatEngine/Resources/Resources.Designer.cs

```diff
@@ -224,7 +222,16 @@ internal static string DeprecatedAdapterPath
                 return ResourceManager.GetString("NonExistingExtensions", resourceCulture);
             }
         }
-
+        
+        /// <summary>
+        ///   Looks up a localized string similar to No test is available for testcase filter `{0}` in {1}.
```

Comment:
`No test is available` or `No tests are available`?

Reply:
`No test is available` has been used in already existing resource string. To keep it consistency keeping it as it is.